pixelpact/src/render.test.js

```diff
@@ -1,6 +1,43 @@
-import { ContentServer } from "./render.js";
+import { ContentServer, BrowserRenderer } from "./render.js";
 import fs from "fs";
 
+describe("BrowserRenderer", () => {
+  let renderer = undefined;
+
+  beforeEach(async () => {
+    renderer = new BrowserRenderer();
+  });
+
+  afterEach(async () => {
+    await renderer.close();
+  });
+
+  it("start starts a browser", async () => {
+    await renderer.start();
+
+    expect(renderer.browser.isConnected()).toBe(true);
+  });
+
+  it("screenshot takes a screenshot of the given url", async () => {
+    await renderer.start();
+
+    const screenshot = await renderer.screenshot("https://ergon.ch");
```

Comment:
This test will fail if `ergon.ch` website is down.

Reply:
Yes, I'm not 100% happy with this test case, but since the renderer accepts an URL, this was the easiest thing to get it to green. The safe way to do it would be to spin up a local server and use this url instead - however, this would make the test much more complex. I think for now, it's fine to ignore this, ok?